package.json

```diff
@@ -1,43 +1,53 @@
 {
-  "name": "my-app",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@testing-library/jest-dom": "^5.16.1",
-    "@testing-library/react": "^12.1.2",
-    "@testing-library/user-event": "^13.5.0",
-    "@types/jest": "^27.4.0",
-    "@types/node": "^16.11.19",
-    "@types/react": "^17.0.38",
-    "@types/react-dom": "^17.0.11",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-scripts": "5.0.0",
-    "typescript": "^4.5.4",
-    "web-vitals": "^2.1.3"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  }
+    "name": "my-app",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "@mui/material": "5.2.8",
+        "@testing-library/jest-dom": "5.16.1",
+        "@testing-library/react": "12.1.2",
+        "@testing-library/user-event": "13.5.0",
+        "@types/jest": "27.4.0",
+        "@types/node": "16.11.19",
+        "@types/react": "17.0.38",
+        "@types/react-dom": "17.0.11",
+        "i18next": "21.6.6",
+        "react": "17.0.2",
+        "react-dom": "17.0.2",
+        "react-i18next": "11.15.3",
+        "react-scripts": "5.0.0",
+        "redux": "4.1.2",
+        "redux-thunk": "2.4.1",
+        "sass": "1.48.0",
+        "typescript": "4.5.4",
+        "web-vitals": "2.1.3"
+    },
+    "scripts": {
+        "start": "react-scripts start",
+        "lint": "eslint src --ext .ts,.tsx",
+        "build": "react-scripts build",
+        "test": "react-scripts test",
+        "eject": "react-scripts eject"
+    },
+    "browserslist": {
+        "production": [
+            ">0.2%",
+            "not dead",
+            "not op_mini all"
+        ],
+        "development": [
+            "last 1 chrome version",
+            "last 1 firefox version",
+            "last 1 safari version"
+        ]
+    },
+    "devDependencies": {
+        "eslint-config-airbnb": "19.0.4",
+        "eslint-config-prettier": "8.3.0",
+        "eslint-plugin-import": "2.25.4",
+        "eslint-plugin-prettier": "^4.0.0",
```

Comment:
You can use a command (`npm config set save-prefix='`') to remove auto-prefixing, then you don't have to do it manually anymore.